os/arch/arm/src/amebad/Kconfig

```diff
@@ -91,6 +91,10 @@ if AMEBAD_BLE_CENTRAL && AMEBAD_BLE_PERIPHERAL
 		default n
 endif
 
+config BLE_MTU
+	int "BLE MTU size"
+	default 240
+
```

Comment:
I think configuring the MTU size have to be decided to platform side not vendor side.

Reply:
Ok, I will fix it. We can set it in ble client config